### PR TITLE
update accounting sdk to latest spec v1.0.2

### PR DESCRIPTION
**Pull request by leewang0**
## Description of the change

* Adds Expenses api
* Adds *LineItems models and properties

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [ ] The code changed/added as part of this pull request has been covered with tests, or the description above explains how testing was performed

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or gif attached. Download "Kap" from the Mac App store for easy gif screen capture.

### CODE REVIEW

File: MergeAccountingClient/model_utils.py [hunk 1]
```diff
@@ -103,59 +103,61 @@ def composed_model_input_classes(cls):
...
-            raise ApiTypeError(
-                error_msg,
-                path_to_item=path_to_item,
-                valid_classes=(str,),
-                key_type=True
-            )
+    path_to_item = []
+    if self._path_to_item:
+        path_to_item.extend(self._path_to_item)
+    path_to_item.append(name)
```
Comment (dan-merge) [inline MergeAccountingClient/model_utils.py]: are these changes intentional, or due to a generator update?

Comment (leewang0) [reply MergeAccountingClient/model_utils.py]: Oh wow great catch, I think I might have an error in one of my templates that overrides the base openapi generator template.

**Review by dan-merge — APPROVED** [review]
